### PR TITLE
gmsh: update to 4.0.5

### PR DESCRIPTION
**Pull request by luzpaz**
###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

Not tested

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: science/gmsh/Portfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ PortGroup           muniversal     1.0
 
 name                gmsh
 conflicts           gmsh-devel
-version             4.0.4
+version             4.0.5
 revision            1
```
Comment (pmetzger) [inline science/gmsh/Portfile]: The revision line should be removed, as this is no longer at revision 1.

File: science/gmsh/Portfile [hunk 2]
```diff
@@ -29,8 +29,8 @@ distname            gmsh-${version}-source
 extract.suffix      .tgz
 
 checksums           rmd160  3efe6a2f40181a1a2a69d8ec558c61d10e1d37ee \
-                    sha256  0a4269a133b6c23a3fca5d3b381d73117ea073b3fbb2c867327677df87a679c3 \
-                    size    12765988
+                    sha256  194354f5cfede4ef47cb6f1d2bde294873315a1b34be114564defef88d108adf \
+                    size    12794616
```
Comment (pmetzger) [inline science/gmsh/Portfile]: This cannot possibly be correct, as it is not possible for the rmd160 to have remained identical while the sha256 changed.

Comment (luzpaz) [reply science/gmsh/Portfile]: I've never generated a rmd160 please advise

Comment (luzpaz) [reply science/gmsh/Portfile]: @mf2k I don't have `port` installed on this machine. Can you provide me with the `rmd160` please ? :sweat_smile:

Comment (mf2k) [reply science/gmsh/Portfile]: The guide also tells you how to generate it. See section 12. 

https://guide.macports.org/#development.creating-portfile

Comment (pmetzger) [reply science/gmsh/Portfile]: If you don't have "port" on your machine, you can't possibly test what you're submitting to us. That's not good.

**Comment by MarcusCalhoun-Lopez** [comment]
Thank you for the notification of the update.
I will push your update with the correct checksums.
For future reference, when you add a modification such as the revision change, it would be best to [squash the changes](https://git-scm.com/book/en/v2/Git-Tools-Re…).

**Comment by pmetzger** [comment]
@luzpaz You can do a pull request at will. It will likely be accepted if you do it correctly.

**Pull request closed without merging**